src/main/asciidoc/_chapters/offheap_read_write.adoc

```diff
@@ -0,0 +1,146 @@
+////
+/**
+ *
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+////
+
+[[offheap_read_write]]
+= RegionServer Offheap Read/Write Path
+:doctype: book
+:numbered:
+:toc: left
+:icons: font
+:experimental:
+
+[[regionserver.offheap.overview]]
+== Overview
+
+For reducing the Java GC impact to P99/P999 RPC latency, HBase 2.x has made the offheap read and write path. The KV are
+allocated from the JVM offheap memory area, which won’t be garbage collected by JVM and need to be deallocated explicitly by
+upstream callers. On the write path, the request packet received from client will be allocated offheap and retained
+until those key values are successfully written to the WAL log and Memstore. The ConcurrentSkipListSet in Memstore does
```

Comment:
key values -> Cells

Reply:
OK,  will replace all the kv/kvs with Cell/Cells. Thanks.